.travis.yml

```diff
@@ -18,7 +18,7 @@ services:
   - mongodb
 
 before_install:
-  - docker build -t xsnippet_api -f contrib/docker/Dockerfile .
+  - docker build -t xsnippet_api -f Dockerfile .
```

Comment:
I don't think there's much sense to explicitly specify `-f Dockerfile` 'cause it's default. Though, doesn't matter much. :)

Reply:
Just a thought: what if we would use `tox -e docker-build` here?